sdk/servicebus/azure-servicebus/MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 include *.rst
 include azure/__init__.py
 recursive-include tests *.py
-recursive-include examples *.py
\ No newline at end of file
+recursive-include samples *.py
```

Comment:
Do we need to account for this "subpackage" in setup.py? (i.e. an exclusion for the wheel)

Reply:
I check the setup.py in servicebus and find 
```
    packages=find_packages(exclude=[
        'tests',
        'examples',
        # Exclude packages that will be covered by PEP420 or nspkg
        'azure',
    ]),
```

Is it here we need to update?

Reply:
Given that EH et al. denote 'samples' in that exclude, I think you have it right Adam.

Thanks for the callout Anna :)